variables.tf

```diff
@@ -48,6 +48,12 @@ variable "subnets" {
   default     = []
 }
 
+variable "network_type" {
+  description = "The type of network stack to use (IPV4 or DUAL)"
+  type        = string
+  default     = "IPV4"
```

Comment:
```suggestion
  default     = null
```

Reply:
This has been changed.  I'll leave it up to your review to mark this conversation as resolved.